### PR TITLE
Adding work section

### PR DESCRIPTION
**Pull request by aron-helu**
A web page of the work section with the projected image tells the languages used to build with and links to the project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,70 +11,180 @@
...
+                        <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
+                        <li><img src="./image/bootstrap.svg" alt="bootstrap-icon"></li>
+                        <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
+                        <li><img src="./image/Ruby.svg" alt="ruby-icon"></li>
+                    </ul>
+                
+                    <button type="button" class="see-project">See Project</button>
+                </div>
+            </section>
         </div>
```
Comment (ShahierNashaat) [inline index.html:187]: - [ ] Kindly be noticed that the works container of the section is a `section` tag.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @aron-helu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aron-helu** [comment]
changing container for the works section and a section tag

**Review by omarsalem7 — APPROVED** [review]
Hi @aron-helu, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors.
✅ Descriptive pull request 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Code Review by @omarsalem7

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**